docs/design/add-in-commands.md

```diff
@@ -130,7 +130,7 @@ Add-in commands are currently supported on the following platforms, except for l
 - Office on the web
 - Office on Windows (Version 1604 (Build 6769.2000) or later, connected to a Microsoft 365 subscription)
```

Comment:
If we are not supporting subscription versions that are past SAC Extended, then should we be mentioning them? Should this be changed to the following?
```suggestion
- Office on Windows connected to a Microsoft 365 subscription
```